lunar/distribution.yaml

```diff
@@ -59,6 +59,22 @@ repositories:
       url: https://github.com/ros/genmsg.git
       version: indigo-devel
     status: maintained
+  gennodejs:
+    doc:
+      type: git
+      url: https://github.com/RethinkRobotics-opensource/gennodejs.git
+      version: kinetic-devel
+    release:
+      tags:
+        release: release/lunar/{package}/{version}
+      url: https://github.com/RethinkRobotics-release/gennodejs-release.git
+      version: 2.0.1-0
+    source:
+      test_pull_requests: true
```

Comment:
Excellent, thanks for that resource! I just added the webhook for issue_comment and pull_request, so we should be good to go!